client/client-multi/client-c/src/kaa/kaa_platform_protocol.c

```diff
@@ -153,126 +164,159 @@ static kaa_error_t kaa_client_sync_get_size(kaa_platform_protocol_t *self,
     return KAA_ERR_NONE;
 }
 
-
-
-static kaa_error_t kaa_client_sync_serialize(kaa_platform_protocol_t *self
-                                           , const kaa_extension_id services[]
-                                           , size_t services_count
-                                           , char* buffer
-                                           , size_t *size)
+// TODO: make buffer uint8_t
```

Comment:
Don't agree. We can't change _all_ buffer in one go, so we should do this gradually along other tasks.